tests/test_wps_parameters.py

```diff
@@ -0,0 +1,11 @@
+from pkg_resources import resource_filename
+import pytest
+
+from wps_tools.testing import run_wps_process
+from osprey.processes.wps_parameters import Parameters
+
+
+@pytest.mark.parametrize(("config"), [resource_filename(__name__, "data/samples/sample_parameter_config.cfg")])
```

Comment:
It would be nice to include a test case for opendap as well.

Reply:
Like, file paths inside the config file are opendap? I think I can make that work, but I'd have to delete a statement from `rvic/parameters.py` which I'm not sure is needed.

Reply:
Could you elaborate more about deleting a statement in `rvic/parameters.py`? Which line are you referring to?

Reply:
So, unlike in `rvic/convolution.py`, `rvic/parameters.py` [calls copy_inputs](https://github.com/UW-Hydro/RVIC/blob/5987da6d33cea8063c848b60215353915ac2bb10/rvic/parameters.py#L167) to copy the inputs to the `CASEDIR`. In doing so, it [calls the copyfile](https://github.com/UW-Hydro/RVIC/blob/5987da6d33cea8063c848b60215353915ac2bb10/rvic/core/utilities.py#L243) function for each file path in the config file. Since the opendap urls don't exists as local files, this won't work. If I get rid of the initial call to `copy_inputs`, this won't be a problem, but I'd be removing functionality. The thing is, since this isn't done in `rvic/convolution.py`, I don't know how necessary this statement is.

Reply:
Hmm, that is very interesting. So none of `POUR_POINTS, UH_BOX, ROUTING, and DOMAIN` supports opendap input? I think at least DOMAIN has to be taken from the server for proper use.

Reply:
From my experience, only POUR_POINTS should be taken from user's local directory. What do you think @corviday ?